requirements/edx/testing.in

```diff
@@ -36,7 +36,7 @@ pyquery                   # jQuery-like API for retrieving fragments of HTML and
 pysqlite                  # DB-API 2.0 interface for SQLite 3.x (used as the relational database for most tests)
 pytest==3.6.3             # Testing framework # Pinned due to https://github.com/pytest-dev/pytest/issues/3749
 pytest-attrib             # Select tests based on attributes
-pytest-cov                # pytest plugin for measuring code coverage
+pytest-cov==2.5.1         # pytest plugin for measuring code coverage
```

Comment:
OK, let's add a comment about why there's a pin, and then we'll be ready to merge.
```
# Pinned due to https://github.com/pytest-dev/pytest-cov/issues/222
```

Reply:
Added and tested by jenkins :)
@nedbat